modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-multiple-bsl-use-case_{context}"]
+= {oadp-short} use case for two BSLs
+
+In this use case, you configure the DPA with two storage locations by using two cloud credentials. You backup a `MySQL` database application by using the default BSL. {oadp-short} stores the backup resources in the default BSL. You then backup the `MySQL` application again by using the second BSL.
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must configure two backup storage locations, AWS S3 and Multicloud Object Gateway (MCG).
+* You must have a `MySQL` database application deployed on a Red Hat OpenShift Cluster.
```

Comment:
@weshayutin I am afraid users can get confused here

should we be less explicit here and tell users can tests with any app? 

or provide an example mysql app yaml to use?

Reply:
I agree, if we're going to walk a customer through a generic example it's probably best to refer to it generically as an application with a database named something like "acme-widgets" .   There are several well known generic k8s sample applications, I would also be fine with pointing customers to devel's sample applications [1]

[1] https://github.com/openshift/oadp-operator/tree/master/tests/e2e/sample-applications

Reply:
Thank you for this clarification. We usually avoid external links in our docs (only link to _our_ docs where needed), for the reason that we have no control over external links. I will use a generic name then instead of mysql.

Reply:
done.